API-strategie-extensies/ext-geospatial.md

```diff
@@ -123,18 +136,66 @@ The guiding priciples for CRS support:
+  <p class="rulelab"><strong>API-40</strong>: Pass the coordinate reference system (CRS) of the request as a parameter</p>
+  <p>Support the <a href="http://docs.opengeospatial.org/is/18-058/18-058.html#_parameter_bbox_crs">OGC API Features part 2 <code>bbox_crs</code> parameter</a> in conformance to the standard.
+  </p>
+  <p>Additionally, if other types of geospatial filters are supported in the API besides <code>bbox</code>: </p>
+  <p>Support the <a href="http://docs.ogc.org/DRAFTS/19-079r1.html#filter-filter-crs">OGC API Features part 3 <code>filter_crs</code> parameter</a> in conformance to the standard.
+  </p>
+</div>
+
+<div class="rule" id="api-41">
+  <p class="rulelab"><strong>API-41</strong>: Pass the desired CRS of the response as a parameter</p>
+  <p>Support the <a href="http://docs.opengeospatial.org/is/18-058/18-058.html#_parameter_crs">OGC API Features part 2 <code>crs</code> parameter</a> in conformance to the standard.
+  </p>
+</div>
+
+<div class="rule" id="api-42">
+  <p class="rulelab"><strong>API-42</strong>: Assert the CRS used in the response using a header</p>
+  <p>Support the <a href="http://docs.opengeospatial.org/is/18-058/18-058.html#_coordinate_reference_system_information_independent_of_the_feature_encoding">OGC API Features part 2 <code>Content-Crs</code> header</a> in conformance to the standard.
+  </p>
 </div>
 
-The following headers are purely meant for negotiation between the client and the server. Depending on the application, the request not only contains geometries but also specific meta data, e.g. the original realistion including the collection date.
+The API should be able to handle the following scenarios based on the rules stated above: 
+
+| Scenario                                        | Explanation |
+| ----------------------------------------------- | ----------- |
+| No geometry in request, no geometry in response | No CRS negotiation necessary |
+| No geometry in request, geometry in response    | The client can request a specific CRS for the geometries in the response using the `crs` parameter. The server indicates the geometry CRS in the response using the `content-crs` header. | 
+| Geometry in request, no geometry in response    | The client indicates the CRS of the request geometry using the `bbox-crs` parameter if a bounding box is used to filter geospatially, or the `filter-crs` parameter if another way of geospatial filtering is used.|
+| Geometry in request, geometry in response       | The client indicates the CRS of the request geometry using `bbox-crs` or `filter-crs` as in the previous scenario, and requests a specific CRS for the geometries in the response using the `crs` parameter. The server indicates the geometry CRS in response using the `content-crs` header.|
```

Comment:
De geometrie in een request is hier vanuit het opvragen van objecten (GET) beschreven, waarbij de geometrie een filter is. Hier zijn `bbox-crs` en `filter-crs` logische parameters.

Wat als ik een object met geometrie (bijvoorbeeld een pand) wil opvoeren (POST) of muteren (POST/PUT), hoe geef ik dan aan wat de CRS van het pandvlak is? Waarschijnlijk mis ik iets of lees ik er overheen, maar ik wil graag weten hoe dat werkt.